### PR TITLE
Translate Cells.rst to [PT]

### PR DESCRIPTION
**Pull request by lira92**

### CODE REVIEW

File: pt/views/cells.rst [hunk 1]
```diff
@@ -1,12 +1,201 @@
...
-    Por favor, sinta-se a vontade para nos enviar um *pull request* para o
-    `Github <https://github.com/cakephp/docs>`_ ou use o botão
-    **IMPROVE THIS DOC** para propor suas mudanças diretamente.
+    *Cell templates* têm um escopo isolado que não compartilha a mesma instância
+    da view utilizada para processar o *template* e o *layout* para o
+    *controller* ou outras *cells*. Assim, eles não sabem de nenhuma chamada de helper
+    feita ou blocos definidos no *template* / layout da *action* e vice-versa.
+    
+Carregando *Cells*
+=============
```
Comment (markstory) [inline pt/views/cells.rst]: Title underlines must be the same length as the text.

File: pt/views/cells.rst [hunk 2]
```diff
@@ -1,12 +1,201 @@
...
+
+.. note::
+
+    O *echo* da *cell* usa o método PHP mágico ``__toString()`` para prevenir o PHP
+    de mostrar o nome do arquivo e o número da linha caso algum erro fatal seja disparado. 
+    Para obter uma mensagem de erro significativa, é remomendado usar o método ``Cell::render()``,
+    por exemplo ``<?= $cell->render() ?>``.
+    
+Renderizando template alternativos
+-----------------------------
```
Comment (markstory) [inline pt/views/cells.rst]: Another underline that is too short.

File: pt/views/cells.rst [hunk 3]
```diff
@@ -1,12 +1,201 @@
 View Cells (Células de Visualização)
 ####################################
 
+View cells são pequenos *mini-controllers* que podem invocar lógica de visualização
+e renderizar templates. A ideia de *cells* é emprestada das `cells do Ruby
+<https://github.com/apotonick/cells>`_, onde desempenham papel e finalidade semelhantes.
+
+Quando usar *Cells*
+=================
```
Comment (markstory) [inline pt/views/cells.rst]: This underline is too short

**Comment by lira92** [comment]
@markstory Can you review again?

**Comment by markstory** [comment]
Thanks @lira92 !

**Comment by lira92** [comment]
:smile:

**Pull request merged**